hazelcast/src/test/java/com/hazelcast/wan/WanReplicationTest.java

```diff
@@ -423,7 +423,12 @@ public Object process(Entry entry) {
 
         @Override
         public EntryBackupProcessor getBackupProcessor() {
-            return null;
+            return MyEntryProcessor.this;
+        }
+
+        @Override
+        public void processBackup(Entry entry) {
+            entry.setValue("TEST");
         }
```

Comment:
is this a fix for a failing test? I don't see an addition into testing part of the code...